src/client/package.json

```diff
@@ -8,6 +8,7 @@
     "react-dom": "^16.2.0",
     "react-fontawesome": "^1.6.1",
     "react-redux": "^5.0.6",
+    "react-router-dom": "^4.2.2",
```

Comment:
You need to also install -S react-transition-group in the client

<img width="785" alt="screen shot 2018-01-28 at 12 20 31 pm" src="https://user-images.githubusercontent.com/9269522/35486835-b08d0d12-0428-11e8-9d0e-4606c935a405.png">